src/script/calling/CallingRepository.ts

```
@@ -2031,7 +2031,7 @@ export class CallingRepository {
       const user = this.userRepository.findUserById(userId);
       if (user) {
         participant = new Participant(user, remoteClientId);
-        call.addParticipant(participant);
+        call?.addParticipant(participant);
```

added to silence an error with call being undefined.